Example/Tests/UnitTests.xctestplan

```diff
@@ -0,0 +1,34 @@
+{
+  "configurations" : [
+    {
+      "id" : "7CD69B1F-AAC5-4B28-B048-0799F574A991",
+      "name" : "Configuration 1",
```

Comment:
What is configuration 1 - can we name it something more descriptive?

Reply:
Renamed to Default